Changelog July 4 (#500)
CloudCannon/platform-documentation: articles/configure-a-url-input.mdx

These keys configure the appearance and functionality of URL inputs in CloudCannon.

<comp.DataReference>
  <comp.DataReferenceRow label="_inputs.*.options.empty_type" type_markdown="string">
    This key determines how CloudCannon handles an empty value. This key does not apply to existing empty values.

    Value must be one of the following:

    * `string` - an empty value for this input will be stored as "".
    * `null` - an empty value for this input will be stored as a null value (default). This does not apply to TOML files.
  </comp.DataReferenceRow>
  <comp.DataReferenceRow label="_inputs.*.options.paths" type_markdown="Object">

>     This key enables you to define paths for your Rich Text editors or File inputs. The following nested keys are available:

Reviewer: ```suggestion
    This key enables you to define paths for this input. The following nested keys are available:
```

Reviewer 2: updated

    * `dam_static`
    * `dam_uploads`
    * `dam_uploads_filename`
    * `static`
    * `uploads`
    * `uploads_filename`
    * `uploads_use_relative_paths`

>     This key has no default. If undefined at higher levels of the [configuration cascade](/documentation/articles/using-the-configuration-cascade/), `paths` will default to any values configured in the [CloudCannon configuration file](/documentation/articles/what-is-the-cloudcannon-configuration-file/).

Reviewer: I feel like this says "If not configured in the cascade, this will default to the configured value in the cascade". Maybe I'm reading this wrong, or it needs a rewrite?

>     For more information, please read our documentation on [Rich Text editors](/documentation/articles/what-are-rich-text-editors/) or [File inputs](/documentation/articles/file-inputs/).

Reviewer: I'm not sure these links are super related? Maybe /documentation/articles/adjusting-the-uploads-path/ would be a better reference?

Reviewer 2: updated

      In this example, we have a `document_url` URL input where users can only enter URLs manually and cannot link to files or upload new files.

      <comp.MultiCodeBlock language="YAML" translate_into={["json"]} source="cloudcannon.config.yml">
      ``````````
      _inputs:
        document_url:
          type: url
          label: Document URL
          comment: Enter the URL to an external document
          options:
            hide_link_to_file: true
      ``````````
      </comp.MultiCodeBlock>
    </comp.Example>
  </comp.DataReferenceRow>
  <comp.DataReferenceRow label="_inputs.*.options.hide_link_to_page" type_markdown="boolean">
    This key toggles whether the "Link to a page" option for this URL input is visible.

    Setting this key to `true` will hide this option.

>     By default, this key is false (i.e., the options visible).

Reviewer: ```suggestion
    By default, this key is false (i.e., the option is visible).
```

  </comp.DataReferenceRow>
  <comp.DataReferenceRow label="_inputs.*.options.hide_link_to_email_address" type_markdown="boolean">
    This key toggles whether the "Link to an email address" option for this URL input is visible.

    Setting this key to `true` will hide this option.

>     By default, this key is false (i.e., the options visible).

Reviewer: ```suggestion
    By default, this key is false (i.e., the option is visible).
```
